airflow/ui/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -18,7 +18,8 @@
  */
 import { Box, Heading } from "@chakra-ui/react";
 
-import { Health } from "./Health";
+import { Health } from "./Health/Health";
```

Comment:
I guess you can create `Health/index.ts` to have `export { Health } from './Health'` to avoid `./Health/Health` and just `./Health` . Similar change for Stats.